### DISCUSSION TITLE
Switch from Suricata to Snort

### DISCUSSION THREAD

**Discussion by tomjones1977** [Security-Onion-Solutions/securityonion#6409] [General] [answered]
When I set up the grid, I configured Suricata with the ET Open ruleset.  I am in a position to purchase the Snort subscriber ruleset now and would like to change the IDS engine from Suricata to Snort.  Can this be done without a re-install?

**Comment by dougburks** [comment, marked as answer]
Security Onion 2 does not include or support Snort at this time:
https://docs.securityonion.net/en/2.3/about.html?highlight=snort#security-onion

**Reply by EusebioGM** [reply]
Hi,
Are you planning to include SNORT in next releasees of Security Onion?
Maybe do you have any forecast about it?
Thanks in advance!

**Reply by dougburks** [reply]
@EusebioGM Here is the feature request: https://github.com/Security-Onion-Solutions/securityonion/issues/4462. There is no timeline. If you have further questions or problems, please start a new discussion rather than replying to somebody else's discussion (per https://github.com/Security-Onion-Solutions/securityonion/discussions/1720).
